app/javascript/tests/enrollment/steps/ExposureInformation.test.js

```diff
@@ -50,6 +50,7 @@ describe('ExposureInformation', () => {
     expect(wrapper.find(Card.Header).text()).toEqual('Monitoree Potential Exposure Information');
     expect(wrapper.find(Card.Body).exists()).toBe(true);
     expect(wrapper.find(Form).exists()).toBe(true);
+    expect(wrapper.find('#continuous_exposure').at(0).prop('disabled')).toBe(false);
```

Comment:
once the tooltip change is made you can add those checks here too

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1241/commits/de6e8f7a42d55d2a02c10112d137563d716be34b

Reply:
it complains about this but that's strange because I'm only seeing one element on the enrollment page with that id:

```
Method “props” is meant to be run on 1 node. 2 found instead.
```

Reply:
Seeing this:
```
Method “props” is meant to be run on 1 node. 0 found instead
```
when I change it to this:
```
expect(wrapper.find('#continuous_exposure').hostNodes().prop('disabled')).toBe(false);
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1241/commits/332e5b7f6a452b5c0b5175963545d09da3b3799f